src/state/queries/messages/leave-conversation.ts

```diff
@@ -18,12 +14,10 @@ export function RQKEY(convoId: string | undefined) {
 export function useLeaveConvo(
   convoId: string | undefined,
   {
-    onSuccess,
     onMutate,
     onError,
   }: {
     onMutate?: () => void
-    onSuccess?: (data: ChatBskyConvoLeaveConvo.OutputSchema) => void
```

Comment:
Didn't see this option being used at all. Easy enough to add back in if you need it.